### DISCUSSION TITLE
4.5.4 tag?

### DISCUSSION THREAD

**Discussion by nvsnkv** [emgucv/emgucv#633] [Q&A] [answered]
Hello! I see that version 4.5.4 is claimed as most recent on emgucv website and there are even packages in Nuget for 4.5.4.
But for some reason it's not tagged in repo so I can't build it for Linux. Can someone please add the tag to the repo?

**Comment by emgucv** [comment, marked as answer]
Release tagged.

**Reply by nvsnkv** [reply]
Thanks a lot!
